setup.cfg

```diff
@@ -4,7 +4,4 @@ universal = 1
 [flake8]
 max-complexity = 10
 max-line-length = 119
-exclude = .venv
-
-[pycodestyle]
-max-line-length = 119
+exclude = .venv,djstripe/migrations/,tests,docs
```

Comment:
@aleccool213 I totally read the diff wrong. I though that was under the flake8 directive. That looks right – we don’t need docs in migrations, tests, or the doc config. Carry on.